Camera2Video/app/src/main/java/com/example/android/camera2/video/fragments/CameraFragment.kt

```diff
@@ -126,23 +130,17 @@ class CameraFragment : Fragment() {
     private val animationTask: Runnable by lazy {
         Runnable {
             // Flash white animation
-            overlay.foreground = Color.argb(150, 255, 255, 255).toDrawable()
+            fragmentCameraBinding.overlay.foreground = Color.argb(150, 255, 255, 255).toDrawable()
             // Wait for ANIMATION_FAST_MILLIS
-            overlay.postDelayed({
+            fragmentCameraBinding.overlay.postDelayed({
                 // Remove white flash animation
-                overlay.foreground = null
+                fragmentCameraBinding.overlay.foreground = null
                 // Restart animation recursively
-                overlay.postDelayed(animationTask, CameraActivity.ANIMATION_FAST_MILLIS)
+                fragmentCameraBinding.overlay.postDelayed(animationTask, CameraActivity.ANIMATION_FAST_MILLIS)
             }, CameraActivity.ANIMATION_FAST_MILLIS)
         }
     }
```

Comment:
This codes seems to depend on luck. Maybe you should use ViewTreeObserver or doOnPreDraw/doOnLayout? see https://developer.android.com/kotlin/ktx